vipps-qr-api.md

```diff
@@ -77,11 +78,10 @@ Below is an example merchant redirect QR:
 | `Accept` | `image/png` | Returns a URL pointing to a png image with 800x800 size |
 | `Accept` | `text/targetUrl` | Returns the target URL of the QR *|
 
-The `targetUrl` that points to `https://qr.vipps.no` is a shortened URL
+The `qrContent` that points to `https://qr.vipps.no` is a shortened URL
 that will be recognized and opened in the Vipps app when scanned from native camera.
 
-**Please note:** It is possible to get the `targetUrl` of the QR code if you need to
-generate the QR code yourselves. This will require an approval from Vipps before it is used, so we can validate
+**Please note:** It is possible to create your own QR code with the qrContent if you need to. This will require an approval from Vipps before it is used, so we can validate
```

Comment:
```suggestion
**Please note:** It is possible to create your own QR code with the `qrContent`. This will require an approval from Vipps before use, so we can validate
```